src/sheets/storeOrder.js

```diff
@@ -81,6 +81,7 @@ function itemToSheetData(
   const companyOrPersonalFields = order.isCompany ? {
     'Reason': order.reason,
     'Requested by': userJiraId,
+    'Manager': order.manager,
     ...variant === 'wincent' ? {} : {Spinoff: order.spinoff},
```

Comment:
you can (and should) now do this:
```suggestion
    ...variant === 'wincent' ? {} : {Spinoff: order.spinoff, Manager: order.manager},
```

Reply:
as for vacuumlabs variant, I added the Manager column to the production sheet.

Reply:
fixed